test/ExecutionFramework.cpp

```diff
@@ -282,3 +282,21 @@ bool ExecutionFramework::storageEmpty(h160 const& _addr) const
 	}
 	return true;
 }
+
+vector<solidity::frontend::test::LogRecord> ExecutionFramework::recordedLogs() const
+{
+	vector<solidity::frontend::test::LogRecord> logs{};
+	for (size_t logIdx = 0; logIdx < numLogs(); ++logIdx)
```

Comment:
I think for the sake of locality I'd just use `m_evmcHost->recorded_logs.size()`, etc. instead of `numLogs()` and `numTopics()` in this function.